release-notes/3.1/releases.json

```diff
@@ -5,7 +5,7 @@
   "latest-runtime": "3.1.0",
   "latest-sdk": "3.1.100",
   "support-phase": "lts",
-  "eol-date": null,
+  "eol-date": "2022-12-03",
```

Comment:
should you do the same change in https://github.com/dotnet/core/blob/master/release-notes/releases-index.json#L12? I'm not really sure how these json files work.

Reply:
Yes, the index file also needs the change that I have added now. Though, it does not drive the marketing site.

Reply:
So is the marketing site driven by the individual files?